plugins/obs-qsv11/device_directx9.h

```diff
@@ -115,4 +105,4 @@ class CD3D9Device : public CHWDevice {
 	BOOL m_bIsA2rgb10;
 };
 
-#endif // #if defined( _WIN32 ) || defined ( _WIN64 )
\ No newline at end of file
+#endif // #if defined( _WIN32 ) || defined ( _WIN64 )
```

Comment:
This change is unrelated.

Reply:
Similarly, I've removed the addition of the `\n` a the end of this file.